### DISCUSSION TITLE
how to correctly draw -NO2 substituant ?

### DISCUSSION THREAD

**Discussion by isabellenavizet** [jsme-editor/jsme-editor.github.io#37] [Q&A]
Hi,
With JSME, I try to draw the nitrobenzen. My concern is that I construct with benzene ring, then N then put the two oxygen and put one with double bond, this give the correct charge + on N but when I put charge - on the oxygen (to deproton) then a double bond arrive automatically to put N that do not follow the octet rule.
How can we avoid this ?
Is this a bug or do I do something wrong ?
thanks for your help
Isabelle

**Comment by jsme-editor** [comment]
Hello Isabelle,
Both form of the nitro group N(=O)=O and [N+]([O-])=O are correct. If you prefer the polar form, you may add keyword polarnitro to the JSME setup (if you are using the editor locally). Otherwise just use the standard form. See also the JSME FAQ.
Best,
Peter
